cleanup-cluster/action.yaml

```
@@ -11,8 +11,12 @@ runs:
   using: "composite"
   steps:
     - name: Delete cluster
-      shell: bash
-      run: |
+      uses: nick-field/retry@7152eba30c6575329ac0576536151aca5a72780e # v3.0.0
```

https://github.com/nick-field/retry leads to a 404. Did you mean https://github.com/nick-fields/retry?

```suggestion
      uses: nick-fields/retry@7152eba30c6575329ac0576536151aca5a72780e # v3.0.0
```

---

ah yes good catch, ty :pray: